contrib/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM debian:buster-slim
 
-RUN apt update && apt install -y --no-install-recommends \
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
```suggestion
ENV DEBIAN_FRONTEND=noninteractive
```

Reply:
According to https://docs.docker.com/engine/reference/builder/#env ENV is persisted in the resulting image. ARG is not persisted in the final image. That's why I have chosen ARG.